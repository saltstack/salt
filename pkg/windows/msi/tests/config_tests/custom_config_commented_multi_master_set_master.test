properties START_MINION="" CONFIG_TYPE=Custom CUSTOM_CONFIG=tests\config_tests\custom_config_commented_multi_master_set_master.conf MASTER=cli.master
dormant
