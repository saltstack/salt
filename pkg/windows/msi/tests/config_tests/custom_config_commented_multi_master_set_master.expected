# User custom config line 1/3
#master:
#- custom.master1
#- custom.master2
master: cli.master
# User custom config line 2/3
id: custom.minion
# User custom config line 3/3
